apps/storybook/src/Surface.stories.tsx

```diff
@@ -39,3 +50,185 @@ export const Basic: Story<SurfaceProps> = ({ elevation }) => {
+        </Text>
+      </Surface.Header>
+      <Surface.Body>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+          eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+          minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+          aliquip ex ea commodo consequat. Duis aute irure dolor in
+          reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+          culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum
+          dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+          incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+          quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+          commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+          velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+          occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+          mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur
+          adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+          magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+          ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
+          irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+          fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+          sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem
+          ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+          tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+          veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
+          ea commodo consequat. Duis aute irure dolor in reprehenderit in
+          voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur
+          sint occaecat cupidatat non proident, sunt in culpa qui officia
+          deserunt mollit anim id est laborum.
+        </p>
+      </Surface.Body>
+      <Divider />
+      <Button styleType='borderless'>View All</Button>
+    </Surface>
+  );
+};
+
+Overflow.args = { elevation: 1, useLayout: true, isPadded: true };
+
+export const NoPadding: Story<SurfaceProps> = (args) => {
+  const { elevation, useLayout, isPadded } = args;
+
+  const cardStyle = {
+    maxHeight: '300px',
+  };
+  return (
+    <Surface
+      elevation={elevation}
+      style={cardStyle}
+      useLayout={useLayout}
+      isPadded={isPadded}
+    >
+      <style>
+        {`ul { all: unset; display: flex; flex-direction: column; flex: 1; list-style-type: none; }`}
+        {`ul li { padding: var(--iui-size-s); position: relative; }`}
+        {`ul li:not(:first-of-type) { border-top: 1px solid var(--iui-surface-border-color); }`}
```

Comment:
Why not using `Divider` instead of this border?

Reply:
Of course! This has been changed